macosx/FileNameCell.mm

```diff
@@ -20,45 +20,51 @@
 static CGFloat const kPaddingBetweenNameAndFolderStatus = 4.0;
 static CGFloat const kPaddingExpansionFrame = 2.0;
 
-@interface FileNameCell ()
-
-@property(nonatomic, readonly) NSAttributedString* attributedTitle;
-@property(nonatomic, readonly) NSAttributedString* attributedStatus;
-@property(nonatomic, readonly) NSMutableDictionary* fTitleAttributes;
-@property(nonatomic, readonly) NSMutableDictionary* fStatusAttributes;
-
-@end
-
-@implementation FileNameCell
-
-- (instancetype)init
+static NSParagraphStyle* kParagraphStyle()
```

Comment:
I don't believe there are any static C functions with the name starting with `k` across the entirety of Darwin, Cocoa or Cocoa Touch. I thought this prefix was reserved for compile-time constants and I would never expect it to be a function. I suggest we don't break the convention and go with a different name without the `k`, here and everywhere.

Reply:
So, we have the rest of the alphabet, but not `k`? OK, I took `s`, for `static`.

Reply:
Generally I see `k` for variables declared constexpr or const, and whose value is fixed for the duration of the program, and (some projects) are simple types that do not require a runtime destructor.

Reply:
Once we move to Swift, we won't have to worry about naming issues since we'll be able to scope static variables inside a class, or using lambda expressions. Maybe we could declare a C++ namespace since we're in Objective-C++, but not worth the trouble for now.

Reply:
GitHub's been showing a "merging is blocked" message for this PR because this thread was never marked as resolved.

Since @DevilDimon marked as approved downthread, I'm going to mark this as resolved to unblock. @DevilDimon if this is in error please LMK.